### ISSUE TITLE
extract: erroneous "Ontology does not contain" warnings

### ISSUE THREAD

**Issue by turbomam** [ontodev/robot#615]
I created `Ontology/ontologies/animals_robot_transition/ro_denied_term_list.txt` with

```
http://purl.obolibrary.org/obo/RO_0000086 # has quality... OBI?
http://purl.obolibrary.org/obo/RO_0000087 # has role... OBI?
http://purl.obolibrary.org/obo/RO_0000092 # disposition of
http://purl.obolibrary.org/obo/RO_0001000 # derives from... OBI?
```

And `curl`ed myself a fresh copy or `ro.owl`. Then I ran

```$ /C/Program\ Files/Java/jdk1.8.0_201/bin/java -Xms4G -Xmx8G \
-jar /C/robot/robot.jar  extract --individuals include --method BOT --force true \
--verbose --intermediates all --annotate-with-source true \
--input Box\ Sync/obo_owl/ro.owl \
--term-file Turbo-Ontology/ontologies/animals_robot_transition/ro_denied_term_list.txt \
--output Turbo-Ontology/ontologies/animals_robot_transition/ro_denied_term_list.txt.ttl
```

and was told

```
2020-01-31 14:57:34,429 WARN  org.obolibrary.robot.OntologyHelper - Ontology does not contain <http://purl.obolibrary.org/obo/RO_0000092>
2020-01-31 14:57:34,430 WARN  org.obolibrary.robot.OntologyHelper - Ontology does not contain <http://purl.obolibrary.org/obo/RO_0000086>
2020-01-31 14:57:34,430 WARN  org.obolibrary.robot.OntologyHelper - Ontology does not contain <http://purl.obolibrary.org/obo/RO_0001000>
2020-01-31 14:57:34,430 WARN  org.obolibrary.robot.OntologyHelper - Ontology does not contain <http://purl.obolibrary.org/obo/RO_0000087>
2020-01-31 14:57:34,715 WARN  org.obolibrary.robot.CommandManager - Subcommand Timing: extract took 3.756 seconds
```

But I don't think that's true. Not only are the terms present in the input ontology, they were successfully extracted:

```
$ grep -C 3 '<http://purl.obolibrary.org/obo/RO_0000092>' Turbo-Ontology/ontologies/animals_robot_transition/ro_denied_term_list.txt.ttl
###  http://purl.obolibrary.org/obo/RO_0000091
<http://purl.obolibrary.org/obo/RO_0000091> rdf:type owl:ObjectProperty ;
                                            rdfs:subPropertyOf <http://purl.obolibrary.org/obo/RO_0000053> ;
                                            owl:inverseOf <http://purl.obolibrary.org/obo/RO_0000092> ;
                                            rdfs:domain <http://purl.obolibrary.org/obo/BFO_0000004> ;
                                            rdfs:range <http://purl.obolibrary.org/obo/BFO_0000016> ;
                                            <http://purl.obolibrary.org/obo/IAO_0000115> "a relation between an independent continuant (the bearer) and a disposition, in which the disposition specifically depends on the bearer for its existence"@en ;
--


###  http://purl.obolibrary.org/obo/RO_0000092
<http://purl.obolibrary.org/obo/RO_0000092> rdf:type owl:ObjectProperty ;
                                            rdfs:subPropertyOf <http://purl.obolibrary.org/obo/RO_0000052> ;
                                            <http://purl.obolibrary.org/obo/IAO_0000115> "inverse of has disposition" ;
                                            <http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/RO_0002259> ;
```

**Comment by jamesaoverton** [comment]
Did we resolve this on the OBI call today, or is it still an open issue?

**Comment by turbomam** [comment]
No, your team resolved a lot of my ROBOT issues yesterday and last week, but I don't think this has been addressed yet. Since the warnings are erroneous and the requested terms actually are extracted, I guess it's not critical.

**Comment by beckyjackson** [comment]
Sorry for the delay on resolving this.

It looks like this is because these terms are part of an import, not actually declared in the RO file. As long as you don't specify `--imports exclude`, these should still be extracted, so this should not spit out a warning. I'm making a fix for this now, thanks for the catch!

**Comment by jamesaoverton** [comment]
This should be fixed by #625
